resources/schema.json

```diff
@@ -115,14 +115,14 @@
                 "@extensions": { "$ref": "#/definitions/default-mutator-config" },
                 "@default": { "$ref": "#/definitions/default-mutator-config" },
                 "Assignment": { "$ref": "#/definitions/default-mutator-config" },
-                "AssignmentEqual": { "$ref": "#/definitions/default-mutator-config" },
                 "BitwiseAnd": { "$ref": "#/definitions/default-mutator-config" },
                 "BitwiseNot": { "$ref": "#/definitions/default-mutator-config" },
                 "BitwiseOr": { "$ref": "#/definitions/default-mutator-config" },
                 "BitwiseXor": { "$ref": "#/definitions/default-mutator-config" },
                 "Decrement": { "$ref": "#/definitions/default-mutator-config" },
                 "DivEqual": { "$ref": "#/definitions/default-mutator-config" },
                 "Division": { "$ref": "#/definitions/default-mutator-config" },
+                "EqualOrIdenticalAssignment": { "$ref": "#/definitions/default-mutator-config" },
```

Comment:
TBH I'm not a big fan of this change because there is an usability issue: imagine you need to disable this mutation, or tell about it someone. `AssignmentEqual` is so much easier to type that you're probably just type it, but `EqualOrIdenticalAssignment` is more likely to get copy-n-pasted, or typed with a typo, or else.

Reply:
it's not just a renaming, the mutator itself changed here so `AssignmentEqual` would now be incorrect. Maybe it can be two distinct mutators, I'm not sure yet...